### PR TITLE
Cosmin/init project

### PR DESCRIPTION
**Pull request by CosminIonascu**

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```
Comment (gcazaciuc) [inline stories/index.js]: Please move this into an examples folder

File: package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "code"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "author": "Ionascu Cosmin",
+  "license": "ISC",
```
Comment (gcazaciuc) [inline package.json]: license should be MIT

File: package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "version": "1.0.0",
+  "description": "A storybook addon that display any type of code highlighted with Prismjs",
+  "keywords": [
+    "addon",
+    "storybook",
+    "code"
+  ],
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (gcazaciuc) [inline package.json]: create a build task that compiles with babel the src dir into dist

Comment (gcazaciuc) [inline package.json]: Make this field point to dist/index.js

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "author": "Ionascu Cosmin",
+  "license": "ISC",
+  "devDependencies": {
+    "@storybook/react": "^3.2.3",
+    "raw-loader": "^0.5.1",
+    "react": "^15.6.1",
```
Comment (gcazaciuc) [inline package.json]: You should commit also yarn.lock file

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (gcazaciuc) [inline .gitignore]: Create also an .npmignore file ad list node_modules and examples and src folders in there

Comment (CosminIonascu) [inline .gitignore]: solved

File: package.json (hunk 2 above)
Comment (CosminIonascu) [inline package.json]: solved

File: package.json (hunk 3 above)
Comment (CosminIonascu) [inline package.json]: solved

File: stories/index.js (hunk 1 above)
Comment (CosminIonascu) [inline stories/index.js]: Solved

**Pull request merged**